site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,235 @@
++++
+date = "2018-05-26T11:30:00+05:30"
+title = "Citra Progress Report 2018 Q1"
+tags = [ "progress-report" ]
+author = "CaptV0rt3x"
+forum = 0000
++++
+
+It's been a while folks. Since the last report, we have had many new features come to Citra. Some
+notable ones include multiplayer, hardware shader improvements (post GLvtx), a logging system rewrite,
+and the highly coveted camera support. Our continuous integration (CI) systems were optimized as well.
+Apart from these, we have had many more minor features, improvements, and bug fixes. So, without further
+ado let's get right into it:
+
+## citra-qt: Multiplayer Improvements ([here](https://github.com/citra-emu/citra/pull/3444), [there](https://github.com/citra-emu/citra/pull/3481), [this](https://github.com/citra-emu/citra/pull/3489), and [that](https://github.com/citra-emu/citra/pull/3676)) by [jroweboy](https://github.com/jroweboy) and [B3n30](https://github.com/B3n30)
+
+Citra has been able to run many games at playable speeds for a while now. However, it was always lacking
+something which the 3DS has: the ability to play with friends. In order to it make a reality, our developers [jroweboy](https://github.com/jroweboy), [B3n30](https://github.com/B3N30), [Subv](https://github.com/subv),
+and [JayFoxRox](https://github.com/jayfoxrox) worked tirelessly for months to reverse engineer workings
+of the 3DS local wireless system. Thanks to their efforts, Citra now emulates _local wireless multiplayer_
+over the internet. This allows you to play your favorite games with your virtual friends across the world.
+Later [jroweboy](https://github.com/jroweboy), [B3n30](https://github.com/B3n30), and few others
+worked on various feature improvements and bug fixes.
+
+### Features
+Currently, the multiplayer features include:
+
+* We now have multiple servers, maintained by our very own [FlameSage](https://community.citra-emu.org/u/flamesage/summary)
+and many other community members at various locations around the globe, so that you can meet others
+online.
+* You can now create rooms which can hold up to 16 players and even secure them with a password.
+* You can now set a preferred game for your room.
+* Since we now have many number of servers, you can also filter the servers to suit your needs.
```

Comment:
`many number of servers` is grammatically incorrect. Can just say `many servers`?